mrp_brewing/report/report_stock_finished_products.xml

```diff
@@ -38,7 +38,9 @@
                                             <span t-att-style="style" t-field="a.with_context(tz=a.env.user.tz).date"/>
                                         </td>
                                         <td>
-                                            <span t-esc="a.raw_material_production_id.brew_order_name"/>
+                                            <t t-if="a.lot_numbers">
+                                                <span t-esc="a.lot_numbers"/>
+                                            </t>

```

Comment:
I suppose it still display the brew order name as lot number for raw_materiel production ?

Reply:
I'm not sure, since lot_numbers are computed with stock_move.quant_ids.lot_id ...
Would this take into account brew order name as lot_number ?
```
<t t-if="a.raw_material_production_id.brew_order_name">
    <span t-esc="a.raw_material_production_id.brew_order_name"/>
</t>
<t t-if="not a.raw_material_production_id.brew_order_name and a.lot_numbers">
    <span t-esc="a.lot_numbers"/>
</t>
```